src/ess/loki/__init__.py

```diff
@@ -5,6 +5,7 @@
 
 from . import data, general, io
 from .general import default_parameters
+from .workflow import live_workflow
```

Comment:
Can we avoid the import, such that users can still import this module without having to install Beamlime (see my earlier comment)?